### PR TITLE
Engine light setup - #64

### PR DESCRIPTION
**Pull request by masonpackard**
# Setup Engine Light

References issue #64. Basic application status setup for engine light. The json response is setup to check for DB and Twilio availability.  There is a basic test for the status page, as well.

The logic follows the suggestions and information referenced by @dirkkelly @bigfleet .

### CODE REVIEW

File: config.ru [hunk 1]
```diff
@@ -1,10 +1,13 @@
 require './app'
 
+set :database, ENV['DATABASE_URL'] || 'postgres://localhost/[YOUR_DATABASE_NAME]'
```
Comment (bigfleet) [inline config.ru]: This doesn't look like a bad line of code, but it doesn't have anything to do with this request.  See if you can get rid of it.

Comment (masonpackard) [inline config.ru]: Surething.  I believe that was added in to help support Boxen for local development.  Specifically, how Boxen sets Postgres to run on a non-standard port.

File: app/routes/status.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
+module Citygram::Routes
+  class Status < Grape::API
+    version 'v1', using: :header, vendor: 'citygram'
```
Comment (bigfleet) [inline app/routes/status.rb]: When you issue the request to get your status, what does that request look like?  As it turns out, we can't send a header.  Perhaps we can use URL versioning for this specific case?  (It's not really a part of our API.)

Comment (masonpackard) [inline app/routes/status.rb]: Updated Status for engine light to be outside of the API and simply return json.

Request:
http://localhost:5000/status

Results:
{"status":"ok","dependencies":["postgres","twilio"],"updated":1438124453}

**Comment by dirkkelly** [comment]
This is fantastic @masonpackard, do you have a demo environment to see it in action?

@bigfleet maybe we could remote pair on enabling pull request environments at some point?

File: spec/routes/status_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Citygram::Routes::Status do
+  include Citygram::Routes::TestHelpers
+
+  describe 'GET /status' do
+    it 'returns a non-empty body' do
+        stub_request(:get, "https://api.twilio.com/2010-04-01").
```
Comment (dirkkelly) [inline spec/routes/status_spec.rb]: Could we abstract this to an environment variable? It's also used [here](https://github.com/codeforamerica/citygram/pull/194/files#diff-7c7d0096272b7f766c0b8676099c7b08R17)

Comment (masonpackard) [inline spec/routes/status_spec.rb]: Refactored to utilize twilio url from env variables.

This has been demo'd in a development environment, but not a "demo" environment at this point.

"GET /status HTTP/1.1" 200

**Here is sample output:**

{"status":"ok","dependencies":["postgres","twilio"],"updated":1438268117}

**Pull request merged**